databay/link.py

```diff
@@ -52,19 +48,20 @@ class Link():
     def __init__(self, 
                  inlets: Union[Inlet, List[Inlet]],
                  outlets: Union[Outlet, List[Outlet]], 
-                 interval:datetime.timedelta, 
+                 interval: Union[datetime.timedelta, int, float], 
                  name:str='',
                  copy_records:bool=True,
-                 catch_exceptions:bool=False):
+                 catch_exceptions:bool=False):             
         """
         :type inlets: :any:`Inlet` or list[:any:`Inlet`]
         :param inlets: inlets to add to this link
 
         :type outlets: :any:`Outlet` or list[:any:`Outlet`]
         :param outlets: outlets to add to this link
 
-        :type interval: datetime.timedelta
-        :param interval: Frequency at which this link should transfer.
+        :type interval: Union[datetime.timedelta, int, float]
+        :param interval: Frequency at which this link should transfer. Can be expressed as :code:`datetime.timedelta` 
```

Comment:
```suggestion
        :param interval: Frequency at which this link should transfer. Can be expressed as :code:`datetime.timedelta` 
        or as type :code:`int` or :code:`float` which will be coerced explicitly to :code:`datetime.timedelta.seconds`
```

Could this sentence be understood as: 'Both timedelta, as well as int or float, will be coerced to timedelta.seconds'?

Reply:
Yeah I think this is more explicit and obvious to users so great catch 👍

Reply:
Right yeah, but then we don't coerce `datetime.timedelta` to `datetime.timedelta.seconds`, do we? Therefore if that's how this sentence could be read, it is incorrect (I'm being picky, I know 🙈). I guess such coercing is redundant, so it shouldn't cause any trouble, but could this doc cause some confusion if such coercion is described but not executed?

Reply:
Hmm... I can see you made a commit to this doc, but I can't see any changes for some reason so I'm a bit confused - sorry if you already changed it and I just missed it. 

In either case, I'm thinking that a rephrase along these lines could clarify it:

> Can be expressed as as type :code:`int` or :code:`float` (which will be coerced explicitly to :code:`datetime.timedelta.seconds`), or as :code:`datetime.timedelta` directly.

or 

> Expects :code:`datetime.timedelta`. Alternatively, you can provide :code:`int` or :code:`float` which will be coerced explicitly to :code:`datetime.timedelta.seconds`.

Reply:
Yeah not sure what that commit was, seems a little weird. In any case, I like the first rephrase you included the best, it seems succinct enough. Will commit that change along with the corrected comments.

Reply:
`Expects :code:datetime.timedelta. Alternatively, you can provide :code:int or :code:float which will be coerced explicitly to :code:datetime.timedelta.seconds.`